tests/test_routes.py

```diff
@@ -808,7 +808,7 @@ def test_route_sub_criteria_scoring_inaccessible_to_commenters(
         ), "Commenter should receive a 302 to authenticator when trying to access the sub criteria scoring page"
         assert (
             response.location
-            == "https://authenticator/service/user?roles_required=TF_LEAD_ASSESSOR|TF_ASSESSOR"  # noqa
+            == "https://authenticator/service/user?roles_required=TF_LEAD_ASSESSOR%7CTF_ASSESSOR"  # noqa
```

Comment:
is this accidental?

Reply:
No, I think this is an intentional change to URLs from this change in utils: https://github.com/communitiesuk/funding-service-design-utils/commit/2c68a71654699eccebe27b896778ae1f1c29520b

Reply:
okay cool